### ISSUE TITLE
latest v0.107.35 not resolve local dns names

### ISSUE THREAD

**Issue by Just2Confuse** [AdguardTeam/AdGuardHome#6046]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to report a bug and not [ask a question or ask for help](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a)

- [X] I have set up AdGuard Home correctly and [configured clients to use it](https://github.com/AdguardTeam/AdGuardHome/wiki/Clients). (Use the [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) for help with installing and configuring clients.)

### Platform (OS and CPU architecture)

Custom (please mention in the description)

### Installation

Other (please mention in the description)

### Setup

On one machine

### AdGuard Home version

v0.107.35

### Action

My local AdGuard get an update today to v0.107.35. Since that time the local DNS resolve dosent work. I see only client IPs and not Names. 

Im on DietPie (Raspi Zero 2W) and bevor this update everything looks fine

### Expected result

DNS Names instead pure IP Adresses

### Actual result

IP Adresses. 

### Additional information and/or screenshots

![adguard](https://github.com/AdguardTeam/AdGuardHome/assets/140651588/d9e077c5-94ea-4a09-a9b9-7eaa848e8212)

```
root@DietPi:~# uname -a
Linux DietPi 6.1.21-v8+ #1642 SMP PREEMPT Mon Apr  3 17:24:16 BST 2023 aarch64 GNU/Linux
root@DietPi:~#
root@DietPi:~# /mnt/dietpi_userdata/adguardhome/AdGuardHome --version
AdGuard Home, version v0.107.35
root@DietPi:~#
```

**Comment by sleel** [comment]
Having the same problem

**Comment by TheCeet** [comment]
I guess eveyone has this issue now.
https://www.reddit.com/r/Adguard/comments/15as5az/adguard_home_is_unable_to_resolve_the_hostname_of/

My setup
Pi 4B, Diepi 64bit, Debian 12
Only issues after update to 0.107.35

**Comment by foxbludd** [comment]
Same here on Unraid, only since last update

**Comment by cenk** [comment]
Same on v0.108.0-a.633+5f8fa006

**Comment by EugeneOne1** [comment]
@Just2Confuse, hello and thanks for the report.  We're actively investigating this, could you please help us providing some additional info?  We'd like to check:

 *  Your [verbose log](https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#verboselog), ideally with private resolving captured;
 *  A screenshot of your private RDNS settings section.

Since these may contain sensitive info, you may sent it to our devteam@adguard.com.

**Comment by Just2Confuse** [comment]
Via Mail? This file wont grow. I also have Unbound, but if needed i can disable it. But it works till yesterday. 

`root@DietPi:/mnt/dietpi_userdata/adguardhome# cat /tmp/aghlog.txt
2023/07/27 12:58:51.618458 3272#6 [info] AdGuard Home, version v0.107.35
2023/07/27 12:58:51.619049 3272#6 [debug] current working directory is /mnt/dietpi_userdata/adguardhome
2023/07/27 12:58:51.619224 3272#6 [info] AdGuard Home is running as a service
2023/07/27 12:58:51.619565 3272#6 [debug] reading config file: /mnt/dietpi_userdata/adguardhome/AdGuardHome.yaml
2023/07/27 12:58:51.626425 3272#6 [debug] github.com/AdguardTeam/AdGuardHome/internal/home.upgradeConfig(): got schema version 24
2023/07/27 12:58:51.626731 3272#6 [debug] reading config file: /mnt/dietpi_userdata/adguardhome/AdGuardHome.yaml
2023/07/27 12:58:51.633419 3272#6 [info] tls: using default ciphers
2023/07/27 12:58:51.636041 3272#6 [debug] hosts container: starting
2023/07/27 12:58:51.636331 3272#6 [debug] hosts container: refreshing
2023/07/27 12:58:51.637033 3272#6 [debug] hosts container: added ip-host pair "127.0.0.1"-"localhost"
2023/07/27 12:58:51.637514 3272#6 [debug] hosts container: added ip-host pair "127.0.1.1"-"DietPi"
2023/07/27 12:58:51.638299 3272#6 [debug] hosts container: sending upd
2023/07/27 12:58:51.646749 3272#6 [debug] filtering: initialized 76 services
2023/07/27 12:58:51.648762 3272#6 [info] safesearch default: disabled
2023/07/27 12:58:51.649260 3272#6 [debug] dhcpd: warning: creating dhcpv4 srv: dhcpv4: invalid IP is not an IPv4 address
2023/07/27 12:58:51.651987 3272#6 [debug] clients: removed 0 client aliases
2023/07/27 12:58:51.652283 3272#6 [debug] clients: added 0 client aliases from dhcp
2023/07/27 12:58:51.652863 3272#6 [debug] writing config file "/mnt/dietpi_userdata/adguardhome/AdGuardHome.yaml"
2023/07/27 12:58:51.652875 3272#12 [debug] clients: removed 0 client aliases
2023/07/27 12:58:51.653254 3272#12 [debug] clients: added 127.0.1.1 -> "DietPi" [1]
2023/07/27 12:58:51.653464 3272#12 [debug] clients: added 127.0.0.1 -> "localhost" [2]
2023/07/27 12:58:51.653828 3272#12 [debug] clients: added 2 client aliases from system hosts file
2023/07/27 12:58:51.688217 3272#6 [info] Initializing auth module: /mnt/dietpi_userdata/adguardhome/data/sessions.db
`

![adguard_scree](https://github.com/AdguardTeam/AdGuardHome/assets/140651588/2ae30674-2dbc-4dd9-8f30-10551e77eae6)

**Comment by cenk** [comment]
OK. I sent the full verbose log via email. I am also having this problem. I hope the log will be useful.

**Comment by EugeneOne1** [comment]
@Just2Confuse, @cenk, we've received the logs.  Looking through it we may conclude that AdGuard Home definitely ignores the data from local resolvers for some reason.  Could you please perform the following command and share its output:

```sh
nslookup -debug <some-client-address> <resolver-address>
```

**Comment by Just2Confuse** [comment]
> @Just2Confuse, @cenk, we've received the logs. Looking through it we may conclude that AdGuard Home definitely ignores the data from local resolvers for some reason. Could you please perform the following command and share its output:
> 
> ```shell
> nslookup -debug <some-client-address> <resolver-address>
> ```

Shure i can. My printer is not confindential :D ;) 

```
root@DietPi:~# nslookup -debug 192.168.1.120 192.168.1.1
------------
    QUESTIONS:
        120.1.168.192.in-addr.arpa, type = PTR, class = IN
    ANSWERS:
    ->  120.1.168.192.in-addr.arpa
        name = EPSON4F2504Drucker.
        ttl = 0
    AUTHORITY RECORDS:
    ADDITIONAL RECORDS:
------------
120.1.168.192.in-addr.arpa      name = EPSON4F2504Drucker.

root@DietPi:~#

```

**Comment by cenk** [comment]
```shell
dietpi@ulasdietpi:~$ nslookup -debug 10.0.0.101 10.0.0.15
------------
    QUESTIONS:
        101.0.0.10.in-addr.arpa, type = PTR, class = IN
    ANSWERS:
    ->  101.0.0.10.in-addr.arpa
        name = ulas-j7prime.ulas.
        ttl = 0
    AUTHORITY RECORDS:
    ADDITIONAL RECORDS:
------------
101.0.0.10.in-addr.arpa name = ulas-j7prime.ulas.
```

**Comment by a904055262** [comment]
Same here on openwrt

**Comment by EugeneOne1** [comment]
@Just2Confuse, @cenk, I believe we've got it, we're going to push the fix to the edge channel later today.

**Comment by foxbludd** [comment]
Sorry late back, if you still need logs I will happily post mine however looks like you've got it! Thanks for looking into this so promptly and to everyone sending logs etc. 👍

**Comment by Just2Confuse** [comment]
> 

Im not sure what this edge means. On Dietpi its a little limited ^^

**Comment by TheCeet** [comment]
> > 
> 
> Im not sure what this edge means. On Dietpi its a little limited ^^

It's a pre-release build
Edge > alpha build
There are alse beta build.
and last: Production build (normal release)
See: https://github.com/AdguardTeam/AdGuardHome/blob/master/README.md#test-unstable-versions

**Comment by lutfor-diu** [comment]
having the same issue. in previous version worked fine.

**Comment by Siox0911** [comment]
Ok, this error chain will prevent me from doing further updates. Even updating to .34 was a huge mistake!

**Comment by EugeneOne1** [comment]
@Just2Confuse, hello again.  Didn't [TheCeet's comment](https://github.com/AdguardTeam/AdGuardHome/issues/6046#issuecomment-1653971646) help you install the edge build?  I'd like to help you with that, but I don't have any information on how exactly have you installed the AdGuard Home.

**Comment by exoticprgmr** [comment]
I can confirm the issue after recent update to v0.107.35  via snap.

**Comment by EugeneOne1** [comment]
@exoticprgmr, as I [mentioned](https://github.com/AdguardTeam/AdGuardHome/issues/6046#issuecomment-1653876076), it seems we've fixed it in the latest edge, so could you please try it and check if it works there?

**Comment by exoticprgmr** [comment]
> @exoticprgmr, as I [mentioned](https://github.com/AdguardTeam/AdGuardHome/issues/6046#issuecomment-1653876076), it seems we've fixed it in the latest edge, so could you please try it and check if it works there?

Thank you. Seems to be fixed on v0.108.0-a.635+79306cb4

**Comment by EugeneOne1** [comment]
@Just2Confuse, we're going to close this issue for now as several people have confirmed that it's fixed.  We'll include the fix in the upcoming beta and production releases, but feel free to reopen the issue if the problem persists.
